lib/ui/text.dart

```diff
@@ -480,11 +604,13 @@ class TextStyle {
       return false;
     if (!_listEquals<String>(_fontFamilyFallback, typedOther._fontFamilyFallback))
       return false;
+    if (!_listEquals<FontFeature>(_fontFeatures, typedOther._fontFeatures))
+      return false;
     return true;
   }
 
   @override
-  int get hashCode => hashValues(hashList(_encoded), _fontFamily, _fontFamilyFallback, _fontSize, _letterSpacing, _wordSpacing, _height, _locale, _background, _foreground, _shadows, _decorationThickness);
+  int get hashCode => hashValues(hashList(_encoded), _fontFamily, _fontFamilyFallback, _fontSize, _letterSpacing, _wordSpacing, _height, _locale, _background, _foreground, _shadows, _decorationThickness, _fontFeatures);
```

Comment:
Should we be putting `_shadows`, `_fontFamilyFallback` and `_fontFeatures` in a `hashList(...)`?